libs/payment/api/src/lib/payment-provider/bexio/bexio-payment-provider.ts

```diff
@@ -26,9 +26,11 @@ import {
   ResponseNOK,
   SendingInvoiceError,
   UnknownIntentState,
-  WebhookNotImplementedError
+  WebhookNotImplementedError,
+  PaymentNotFound
 } from './bexio-errors'
 import {addToStringReplaceMap, mapBexioStatusToPaymentStatus, searchForContact} from './bexio-utils'
+import fetch from 'node-fetch'
```

Comment:
why is this imported here? I can't see it used in your changes

Reply:
it was added by me, but starting with node 18 there's no longer need to import fetch from node-fetch (it's included). But I think it's still ok, it can still be imported as a "fallback" if older version of node is used